cypress/integration/visual-test/list-test.spec.js

```diff
@@ -0,0 +1,47 @@
+import { baseURL, checkEyes } from '../../util';
+describe('Visual Regression Testing', () => {
+  before(() => {
+    cy.eyesOpen({
+      appName: 'Clarity',
+      testName: 'list',
+      browser: { width: 1024, height: 768, name: 'chrome' },
```

Comment:
Can we abstract this browser config object so its shared, like you did with `baseUrl`? Then we could easily add a new config globally.

Reply:
done